scale/docs/rest/recipe.rst

```diff
@@ -409,3 +409,259 @@ These services provide access to information about recipes.
+
+
+.. _rest_recipe_reprocess:
+
++-------------------------------------------------------------------------------------------------------------------------+
+| **Reprocess Recipe**                                                                                                    |
++=========================================================================================================================+
+| Creates a new recipe using its latest type revision by superseding an existing recipe and associated jobs.              |
+| Note that if the recipe type definition has not changed since the recipe was created, then one or more job names must be|
+| specified to force the recipe to be reprocessed.                                                                        |
++-------------------------------------------------------------------------------------------------------------------------+
+| **POST** /recipes/{id}/reprocess/                                                                                       |
+|          Where {id} is the unique identifier of an existing model.                                                      |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **Content Type**   | *application/json*                                                                                 |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **JSON Fields**                                                                                                         |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| job_names          | Array[String]     | Optional | The name of jobs within the recipe definition that should be        |
+|                    |                   |          | included in the reprocessing request, even when the definition for  |
+|                    |                   |          | those jobs has not changed between recipe type revisions.           |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| all_jobs           | Boolean           | Optional | A flag that indicates all jobs in the recipe should be reprocessed, |
+|                    |                   |          | even when the recipe type definitions are identical. This option    |
+|                    |                   |          | overrides any job_name parameters and is typically used to re-run   |
+|                    |                   |          | jobs that failed due to temporary errors.                           |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
```

Comment:
I wouldn't say "is typically used to re-run jobs that failed due to temporary errors", as that is the point of re-queuing.